### ISSUE TITLE
Wierd Kerning & Tracking Support?

### ISSUE THREAD

**Issue by C29H25N3O5** [love2d/love#1780]
It seems that the kerning support for the in-game text display is not working very well.

<img width="1035" alt="43AAAC41FA51D24C0C6C80CE5280F14F" src="https://user-images.githubusercontent.com/87485746/154858239-d031e617-65d9-4a30-b56d-313a46a4ea0f.png">

The white font is rendered in the game, and the black text is rendered in Glyphs.

Here is one more example, with the blue text being the one rendered in Pixelmator Pro.

<img width="1112" alt="49E577E8A8276A5AFDB678D297C0838D" src="https://user-images.githubusercontent.com/87485746/154858296-61a3147f-1ff7-4389-899b-2ad30a5610a5.png">

Maybe the tracking (=the normal space between the characters) is too tight here?

**Comment by slime73** [comment]
love uses FreeType's kerning and spacing APIs do determine glyph positioning. Can you post some code and/or the font which demonstrates the issue for you?

**Comment by C29H25N3O5** [comment]
```
local f = love.graphics.setNewFont('26FSans-Regular.otf', 40)
love.graphics.setFont(f)
function love.draw()
    love.graphics.print('AVAWT Hello World ToToTo', 10, 10)
end
```

Here is the result:
![Screenshot 2022-02-20 at 13 07 38 copy](https://user-images.githubusercontent.com/87485746/154859949-2811b35a-2f92-45c6-a08b-19cd3e3c4764.jpg)

Compared with texts rendered in Pixelmator Pro:
<img width="796" alt="F81E9BE36D5CAD4A4B8B744562F4EE8A" src="https://user-images.githubusercontent.com/87485746/154859871-bd90d39b-a114-4e39-b6f9-278f31edfaf4.png">

**Comment by slime73** [comment]
Can you upload a font file we can use for testing?

**Comment by C29H25N3O5** [comment]
https://github.com/26F-Studio/26F-Sans/releases/tag/V1.0.0

**Comment by slime73** [comment]
FreeType doesn't think the font has kerning (although I do see it in FontForge).

I noticed this comment in FreeType's headers, maybe it's related? `FT_FACE_FLAG_KERNING ` is not set, for the font.
```cpp
  /*    FT_FACE_FLAG_KERNING ::                                            */
  /*      The face contains kerning information.  If set, the kerning      */
  /*      distance can be retrieved using the function @FT_Get_Kerning.    */
  /*      Otherwise the function always return the vector (0,0).  Note     */
  /*      that FreeType doesn't handle kerning data from the SFNT `GPOS'   */
  /*      table (as present in many OpenType fonts).                       */
```

**Comment by slime73** [comment]
After reading about it even more here https://freetype.org/freetype2/docs/glyphs/glyphs-4.html I believe this will be solved once we integrate Harfbuzz (#1687).

**Comment by slime73** [comment]
This should be fixed in love 12 now as of [8572d4f](https://github.com/love2d/love/commit/8572d4f9353da812f7d8d428446a38e5064592a2).
